content/docs/setup/kubernetes/install/operator/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Operator Install 
+description: Install and configure Istio using the Istio Operator.
+weight: 25
+keywords: [operator,kubernetes,helm]
+---
+
+Follow this guide to install and configure an Istio mesh using an alternate
+installation method: the Istio {{<gloss operator>}}Operator{{</gloss>}}
+installation.
+
+The Istio Operator installation process has several benefits:
+
+- Common install tasks require fewer steps, some are a simple one-line command.
+
```

Comment:
again, I'd like to see the elevator pitch in a separate document.  We don't yet have such a structure for the feature landing pages, and IMO we need such a structure.  I am not clear how to proceed, but this is the wrong place for "Why Operator?"  @frankbu or @geeknoid can you help find a landing page for our various features that don't involve making the docs look like a sales pitch :)